named_entity_recognition/README.md

```diff
@@ -0,0 +1,35 @@
+# Named Entity Recognition with Kubeflow and Keras 
+
+This example demonstrates how you can use Kubeflow to train and deploy a Keras model with a custom prediction routine. 
+
+## Goals
```

Comment:
Consider changing the wording to something like, "In this walkthrough you will learn how to..."

Reply:
✓ done